MekHQ/src/mekhq/gui/model/UnitTableModel.java

```diff
@@ -236,6 +241,14 @@ public Object getValueAt(int row, int col) {
                 return u.getCommander().getFullTitle();
             }
         }
+        if(col == COL_FORCE) {
+            Force force = u.campaign.getForce(u.getForceId());
```

Comment:
I just noticed that `campaign` is a public field on `Unit`. Interesting.

Reply:
Once the money change is finished, I'll do a PR cleaning Unit and wrap that in get/set calls (I want to avoid possible merge conflicts).

Reply:
No worries, I've got it handled already.

Reply:
Thanks!